lib/event-parser.js

```diff
@@ -2,10 +2,34 @@
 
 const EventParserError = require('./event-parser-error');
 
+const ModelEvent = require('./model-event');
+
 class EventParser {
+	/**
+	 * Get the event actions
+	 *
+	 * @param {String} message object
+	 * @return {Array} Event actions
+	 * @throw EventParserError
+	 */
+	static async getSubscribers(message) {
+		if(!message || typeof message !== 'object')
+			throw new EventParserError('message is required', EventParserError.codes.INVALID_DATA);
```

Comment:
```suggestion
			throw new EventParserError('Message is required and must be an object', EventParserError.codes.INVALID_DATA);
```

Por otro lado, si es un array pasa tu validación. Deberías validar también con `Array.isArray(message)`

Reply:
Agregar caso de prueba a los tests un `message` que sea un array.